### PR TITLE
Update adding-outside-collaborators-to-repositories-in-your-organizat…

### PR DESCRIPTION
**Pull request by gennaropalma**
…ion.md

Expand a little about OC bypassing SAML SSO, and not being available for EMU. Currently, the only mentions are sort of buried in linked documents.

### Why:

Closes: 

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: content/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ Before you can add someone as an outside collaborator on a repository, the perso
 If your organization requires two-factor authentication, all outside collaborators must enable two-factor authentication before accepting your invitation to collaborate on a repository. For more information, see "[AUTOTITLE](/organizations/keeping-your-organization-secure/managing-two-factor-authentication-for-your-organization/requiring-two-factor-authentication-in-your-organization)."
 {% endif %}
 
+{% ifversion not ghae %}
+Outside collaborators bypasses SAML SSO requirements. For more information, see "[AUTOTITLE](https://docs.github.com/en/enterprise-cloud@latest/admin/identity-and-access-management/using-saml-for-enterprise-iam/configuring-saml-single-sign-on-for-your-enterprise)"
+
+Outside collaborators cannot be added to Enterprise Managed Users. For more information, see "[AUTOTITLE](content/admin/identity-and-access-management/using-enterprise-managed-users-for-iam/about-enterprise-managed-users.md)."
+{% endif %}
```
Comment (isaacmbrown) [inline content/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization.md:45]: I think we might want the wording to be more like this? That said, I wonder if we'll be updating this article shortly anyway in light of https://github.com/github/releases/issues/3591. @mattpollard Was that in your plans?

```suggestion
{% ifversion ghec %}
Outside collaborators cannot be added to a repository owned by a {% data variables.enterprise.prodname_emu_org %}. For more information, see "[AUTOTITLE](/admin/identity-and-access-management/using-enterprise-managed-users-for-iam/about-enterprise-managed-users)."
{% endif %}
```

**Review by isaacmbrown — COMMENTED** [review]
Thanks @gennaropalma! Left a couple of comments/questions

**Comment by myarb** [comment]
👋🏼 @gennaropalma Feel free to ping us when this is ready for another review ✨

File: content/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization.md (hunk 1 above)
Comment (gennaropalma) [reply content/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization.md:45]: The proposed change suggests to the reader that the restriction is with adding them to a repository only, when they cannot be added to an EMU enterprise at all. See update below.

**Comment by gennaropalma** [comment]
Hi team 👋 Waiting on some updates regarding the [guest collaborator private beta](https://github.blog/changelog/2022-10-31-private-beta-introducing-innersource-restrictions-for-enterprise-managed-users/) which could make this out of date

File: content/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization.md (hunk 1 above)
Comment (isaacmbrown) [reply content/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization.md:45]: No worries @gennaropalma! In that case maybe we can do something like this. We have docs published for the guest collaborators beta now, so I think we can also add a link to that here.

```suggestion
Outside collaborators cannot be added to an {% data variables.enterprise.prodname_emu_enterprise %}. However, you can grant limited access to users outside your enterprise using the guest collaborator role. For more information, see "[AUTOTITLE](/admin/managing-accounts-and-repositories/managing-users-in-your-enterprise/roles-in-an-enterprise#guest-collaborators)."
{% endif %}
```

**Comment by isaacmbrown** [comment]
Hi @gennaropalma, added another suggestion above, including how we might want to incorporate the guest collaborators reference. Let me know if I can help further!

**Comment by isaacmbrown** [comment]
Hi @gennaropalma, I went ahead and applied a couple of suggestions, so I'll merge this now! Thanks for your contribution 😀

**Review by isaacmbrown — APPROVED** [review]

**Pull request merged**